python/ray/autoscaler/_private/command_runner.py

```diff
@@ -349,7 +349,7 @@ def run(
                         cf.bold(local),
```

Comment:
is this log also the other way around?

Reply:
The log is OK, just the actual command is the wrong way. It only worked if the ports were the same.

Reply:
I think the confusion is from `localhost` which actually means the remote host: https://unix.stackexchange.com/questions/115897/whats-ssh-port-forwarding-and-whats-the-difference-between-ssh-local-and-remot